deploy/kubernetes/overlays/stable-1-21/kustomization.yaml

```diff
@@ -7,4 +7,4 @@ resources:
 patchesStrategicMerge:
 - no-node-driver-registrar-probe.yaml
```

Comment:
Is this still necessary?

Reply:
More background in https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/830#issuecomment-948111518

The feature was introduced in node-driver-registrar in v2.4.0, if +1.21 is already using this version then it would be great to remove this patch to see it working